brave-intl/publishers
# Add Crypto Payment Widget, QR code, and assets

Resolves https://github.com/brave-intl/creators-private-issues/issues/1651
Resolves https://github.com/brave-intl/creators-private-issues/issues/1650
Resolves https://github.com/brave-intl/creators-private-issues/issues/1649

**Testing Instructions:**

log in as [Alice@default.org](mailto:Alice@default.org).  When landing on the publisher’s dashboard, make sure to assign some of the existing crypto addresses to the ‘uphold connected twitter’ channel.  Then you should be able to navigate to https://localhost:3000/c/c6587d7de522444595e2, and see the channel page details and widget.    Press ‘send’ to connect a wallet.  The user can select what network they’re on, so check to make sure you are on a test network.  Faucets for Solana and eth can be found here: https://solfaucet.com/, https://faucet.quicknode.com/ethereum/sepolia
